packages/dappmanager/src/modules/https-portal/migration.ts

```diff
@@ -118,7 +118,8 @@ export function migrateCoreNetworkAndAliasInCompose(
   if (
     composeNetwork?.name === params.DNP_PRIVATE_NETWORK_NAME &&
     composeNetwork?.external &&
-    serviceNetwork.aliases?.includes(alias)
+    serviceNetwork.aliases &&
+    serviceNetwork.aliases.includes(alias)
```

Comment:
This doesn't fix the bug, the new syntax is equivalent to the old one. The issue is that apparently `serviceNetwork` is not properly typed and can be nullish